### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by chinweokwu**
This project is about creating a replica apple page, and new-feature branch is the final branch and the main branch of this project.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @chinweokwu 

### You have done a good job so far, however, there are a few changes you need to make.
- In your navigation, use the background gradient to achieve the design
Screenshot from 2020-02-11 16-46-22
- Write a more descriptive pull request stating what you have worked on the project.
- The spacing between your thumbnails images is big, reduce it
Screenshot from 2020-02-11 16-44-06
- Your footer is still off, refer the original website
Screenshot from 2020-02-11 16-45-36
- Configure stickler well to help with the linting

- Make the changes and resubmit for another code review.
- Happy coding!

- For assistance: Contact me @rOluochKe

**Comment by chinweokwu** [comment]
Hi Raymond,
the images and the size are good on my screen i don't know how to adjust to
fix a bigger screen

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @chinweokwu      
#### Great Job so far, but there are a few more things required. 
### Navigation 
- [ ] Any time a link is hovered over the cursor changes but your links on the navbar don't, that should be fixed.

### Background
- [x] The background image isnt covering the whole page,lets fix that.


### Footer
- [x] Please align the footer links to look like the original.
### Original  Version


### Your  Version 

 - [x] Please lets fix this broken image link



### Make changes and submit another code review request.
*Happy coding*

**Comment by chinweokwu** [comment]
hi,
i have tried to fix the nav link issues. but its no working would you mind
just checking it for me pls

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  background-size: cover;
+  width: 100%;
+  position: absolute;
+  height: 650px;
+  z-index: -1;
+  background-color: rgba(0, 0, 0, 0.9);
+}
+
+nav {
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, .2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, .2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
```
Comment (euqueme) [inline styles.css]: - [ ] Remove the url image from the gradient in line #29 as it has to be achieved using only colors and not images

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  <title>Apple Store</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel="stylesheet" href="https://unicons.iconscout.com/release/v2.0.1/css/unicons.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100i&display=swap" rel="stylesheet">
+  <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body>
+  <!-- Header sec -->
+  <header id="showcase">
+    <div class="bg-img">
```
Comment (euqueme) [inline index.html]: do this change to make your hovers work: delete the div on line #16 (just the div not the content) and pass the styling from the .bg-img class to the #showcase id

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+> This project consists of building an HTML document that matches the appearance of [apple.com’s](https://www.apple.com) appletunes page
+
+![screenshot](./Screenshot.png)
+
+## Built With
+
+- HTML, CSS,
+- Flexbox
+- grid
+-
```
Comment (euqueme) [inline README.md]: Delete the hyphen  `-`  in line #13

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hi @chinweokwu :wave: 

### This is a good effort, :muscle:  I can see you have done some changes required by the previous TSE's, but you have to complete all the changes to get the approval

## Hover Issue

- [x] I have commented on your code with something that worked for me in my computer basically, you don't need the div with the .bg-image class, because it's preventing you from accessing your navbar, that's why you don't see the hover styling (of course you'd have to restyle a little so let's do it! :raised_hands: ).

 ## Building with gradients

- [x] please remove the url image in your gradient as this has to be achieved using only colors

## Footer

- [x] please align the footer links to make it look like the original as it was requested by the previous reviewer

## Generals

- [x] Validate your HTML file using this [validator](https://validator.w3.org/#validate_by_upload)

- [ ] Linters: you have linter errors do not submit back unless you have a green check in the last commit :heavy_check_mark: 

### Thats it! :raised_hands: 

### Please :pray:  do the changes :memo: and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any questions? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work @chinweokwu ! :medal_military: 
Your work looks very exquisite and it came together very nicely :ok_hand: , however, there is one change I have to propose, based on the hard requirements for this project. Please consider it stronlgly! When you have made the appropriate changes, please request another review :handshake: 

#### Use of background images
Please consider that as per the requirements for this project, the images in the footer are supposed to be `<div>` tags with their `background-image` properties set to the image URL and not image tags inside divs
screenshot

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @chinweokwu, you did a great job making changes requested by previous reviewers. 👍 However, we still have left out some requirements. Find comments below: 😄 
- Kindly fix the warning returned by [HTML markup](https://validator.w3.org/)
ap3
 - Currently, the bottom elements are not links but they should be wrapped with ```<a>```
ap4

Kindly make these changes and submit another code review 🙂 
Happy coding 💻 
Daniel Larbi Adoo

**Comment by chinweokwu** [comment]
hi larbi,
can you pls explain what i should do here because i don't understand this
[image: ap4]
<https://user-images.githubusercontent.com/46388113/74503100-c5226900-4ee7-11ea-9a17-d7f8bd76acab.PNG>

**Review by addod19 — APPROVED** [review]
## Status: Approved ✈️ 

Hi @chinweokwu, congratulations on a great job done. 👌 You can go ahead and merge to master and submit project completion form. 😄  Good luck with your next project.

Happy coding 💻 
Daniel Larbi Addo

**Pull request merged**